test/e2e/all_in_one_test.go

```diff
@@ -26,47 +29,41 @@ import (
 
 const TrackingID = "MyTrackingId"
 
-func JaegerAllInOne(t *testing.T) {
-	ctx := prepare(t)
-	defer ctx.Cleanup()
+type AllInOneTestSuite struct {
+	suite.Suite
+}
 
-	if err := allInOneTest(t, framework.Global, ctx); err != nil {
-		t.Fatal(err)
-	}
+func(suite *AllInOneTestSuite) SetupSuite() {
```

Comment:
Does the SDK clean up all resources (namespace..) when no returning an error?

Reply:
@pavolloffay cleanup should happen in the TearDownSuite method where ctx.cleanup() is called.  This should work the same as previous tests did.